Dockerfile

```diff
@@ -65,16 +56,9 @@ RUN \
 RUN \
     groupadd --gid 10001 app && \
     useradd --uid 10001 --gid 10001 --home /app --create-home app && \
-    # first, an apt-get update is required for gnupg, which is required for apt-key adv
+    # first, an apt-get update is required for the packages
```

Comment:
please kill this comment entirely as well (it was explaining the previous need for calling `apt-get update` twice here)